modules/nw-network-observability-operator.adoc

```diff
@@ -0,0 +1,90 @@
+// Module included in the following assemblies:
+// * networking/network_observability/understanding-network-observability-operator.adoc
+
+:_content-type: PROCEDURE
+[id="nw-network-observability-operator_{context}"]
+= Network Observability Operator
+
+When you deploy flow collector, the Operator provides dashboards, metrics, and keeps flows accessible in a queryable log store, Grafana Loki. When a flow collector is deployed, new dashboards are available in the Console.
+
+.Procedure
+
+. Run the following command to view the state of `flowcollector`:
++
+[source,terminal]
+----
+$ oc get flowcollector/cluster
+----
++
+.Example output
+----
+NAME      AGENT   SAMPLING (EBPF)   DEPLOYMENT MODEL   STATUS
+cluster   EBPF    50                DIRECT             Ready
+----
+
+. Check the status of pods running in namespace `netobserv`.
++
+[source,terminal]
+----
+$ oc get pods -n netobserv
+----
++
+.Example output
+----
+NAME                              READY   STATUS    RESTARTS   AGE
+flowlogs-pipeline-56hbp           1/1     Running   0          147m
+flowlogs-pipeline-9plvv           1/1     Running   0          147m
+flowlogs-pipeline-h5gkb           1/1     Running   0          147m
+flowlogs-pipeline-hh6kf           1/1     Running   0          147m
+flowlogs-pipeline-w7vv5           1/1     Running   0          147m
+netobserv-plugin-cdd7dc6c-j8ggp   1/1     Running   0          147m
+----
+
+Flowlogs-pipeline pods collect flows, enriches the collected flows, then send flows to the Loki storage.
+Netobserv plugin pods create a visualization plugin for the {product-title} Console.
+
+. Check the status of pods running in namespace `netobserv-privileged`.
++
+[source,terminal]
+----
+$ oc get pods -n netobserv-privileged
+----
++
+.Example output
+----
+NAME                         READY   STATUS    RESTARTS   AGE
+netobserv-ebpf-agent-4lpp6   1/1     Running   0          151m
+netobserv-ebpf-agent-6gbrk   1/1     Running   0          151m
+netobserv-ebpf-agent-klpl9   1/1     Running   0          151m
+netobserv-ebpf-agent-vrcnf   1/1     Running   0          151m
+netobserv-ebpf-agent-xf5jh   1/1     Running   0          151m
+----
+
+Netobserv eBPF agent pods monitor the nodes network interfaces to get flows and send them to Flowlogs Pipeline pods.
+
+. If you are using a Loki Operator, check the status of pods running in namespace `openshift-operators-redhat`.
```

Comment:
. If you are using a Loki Operator, check the status of pods running in the `openshift-operators-redhat`namespace by entering the following command:

Reply:
addressed